### PR TITLE
Fix Chinese chars encoding on eml files

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-22015

## Description
Fixed an issue where an eml file containing chinese characters was not decoded correctly.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
failed unit-test: https://app.circleci.com/pipelines/github/demisto/parse-emails/300/workflows/273fc841-7667-4e8b-8be2-dd018d8f2e02/jobs/1387

**Review by MLainer1 — APPROVED** [review]
🆒

File: parse_emails/tests/test_data/chinese.eml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Content-Type: text/plain; charset="gb2312"
+MIME-Version: 1.0
+Content-Transfer-Encoding: base64
+Subject: Greeting in Chinese
+From: Sender Name <sender@example.com>
+To: Recipient Name <recipient@example.com>
+Date: Mon, 06 Mar 2023 19:47:06 +0200
```
Comment (JudahSchwartz) [inline parse_emails/tests/test_data/chinese.eml:7]: shouldnt there be chinese chars in this test?

Comment (GuyAfik) [reply parse_emails/tests/test_data/chinese.eml:7]: its encoded in base64, so no.

Comment (GuyAfik) [reply parse_emails/tests/test_data/chinese.eml:7]: @JudahSchwartz ?

**Pull request merged**